README.md

```diff
@@ -3,149 +3,136 @@
-  [parentelement.com/assets/json_net_unity](https://www.parentelement.com/assets/json_net_unity)  
-  [assetstore.unity.com/packages/tools/input-management/json-net-for-unity](https://assetstore.unity.com/packages/tools/input-management/json-net-for-unity-11347)
-
-## But why another solution
-
-> SaladLab and ParentElement, among other unmentioned coders,
-> have done great work in making Json.NET work in Unity. _So why do we need mine?_
->
-> Both mentioned projects are discontinued, and outdated by some margin in result.
-> SaladLab's latest release targets Json.NET 9, and ParentElement's Json.NET 8.
-> We're at Json.NET 12 now! _(at time of writing)_
->
-> Changeset, compared to SaladLabs and ParentElements solutions:
->
-> - Json.NET version 12 _(let's see how long I can keep up with James repo)_
-> - Release available via Unity Package Manager
-> - Unit testing in Unity Editor
-> - Unit testing in IL2CPP built platforms _(ex: StandaloneWindows)_
->
-> A rework is in this case easier than jumping into SaladLabs project to update it. Sorry Salad champ.
->
-> _Although, in the future this may also be old stuff too,
-> with the upcoming new [System.Text.Json](https://devblogs.microsoft.com/dotnet/try-the-new-system-text-json-apis/)
-> namespace and need to keep the project up-to-date with Newtonsoft.Json.
-> But not for now!_
+### Run tests
+
+Run the Newtonsoft.Json.Tests normally via the Test Runner inside Visual Studio.
+
+For testing inside Unity locally, look inside the
+[Src/Newtonsoft.Json-for-Unity.Tests/README.md](/Src/Newtonsoft.Json-for-Unity.Tests/README.md)
+for more information.
+
+### Merging changes from JamesNK/Newtonsoft.Json
+
+As an example, this is how to make a release for Json.<i></i>NET 12.0.3,
+resulting in Newtonsoft.Json-for-Unity (this project) version 12.0.301. Check
+the [About the versioning](https://github.com/jilleJr/Newtonsoft.Json-for-Unity/wiki/About-the-versioning)
+guide on the Wiki about the version format.
+
+1. First, read the [Working with branches][wiki-workingwithbranches] wiki page.
+
+2. If not yet done, create the release branch `release/12.0.301` based off of
+   the `master` branch.
+
+3. Checkout a new feature branch `feature/merge-12.0.3` based off of the
+   `release/12.0.301` branch.
+
+4. Merge the changes from the release tag of JamesNK repo.
+   For some files the changes are too grand and the auto merging fails.
+   Make sure then to compare said files with the actual changeset on JamesNKs
+   repo, like so: <https://github.com/JamesNK/Newtonsoft.Json/compare/12.0.2...12.0.3>
+
+    ```bash
+    git checkout -b feature/12.0.301/merge-12.0.3
+    git pull https://github.com/JamesNK/Newtonsoft.Json.git 12.0.3
+    # Resolve merge conflicts
+    # Only proceed when fully done
+    git commit -m "Merge JamesNK/Newtonsoft.Json:12.0.3 into feature/12.0.301/merge-12.0.3"
+    git push
+    ```
+
+4. Create a merge request from `feature/12.0.301/merge-12.0.3` to `release/12.0.301`  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[ordered-list-marker-value] Marker should be `5`, was `4`](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4743215)